web/libs/editor/src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx

```diff
@@ -199,44 +199,47 @@ const OutlinerInnerTreeComponent: FC<OutlinerInnerTreeProps> = observer(({ regio
   );
 });
 
+export const displayRegionLabel = (item: any) => {
```

Comment:
make this a proper react component maybe? it will look better as an exported resource + won't be worse to use it as component in this file as well. and we actually should have something similar in other places.

Reply:
Done, now it's a component.